### DISCUSSION TITLE
Plugins used in animated windows gif in README.md

### DISCUSSION THREAD

**Discussion by sansarselim** [neovide/neovide#633] [Q&A] [answered]
Hi What plugins used in this gif?
https://github.com/Kethku/neovide/blob/main/assets/AnimatedWindows.gif
I want my split windows change size when switched to. Also it seems word wrap is disabled too and buffers shown on tab bar instead of tabs. Can you share your vimrc used to create that gif. Thanks.

**Comment by Kethku** [comment, marked as answer]
https://github.com/Kethku/vim-config here is my config. Its written in fennel using the aniseed plugin.

plugins are added here: https://github.com/Kethku/vim-config/blob/main/init.vim
basic config is here: https://github.com/Kethku/vim-config/blob/main/fnl/module/settings.fnl
keybindings here: https://github.com/Kethku/vim-config/blob/main/fnl/module/bindings.fnl
